docs/content/tutorial-cmake.md

```diff
@@ -87,6 +88,20 @@ should not be included in the installed version of the library. This prevents
 details of your build from leaking into the install. We'll handle the include
 directory for the installed library differently.
 
+There is one last step for creating the C library. Remember how we passed
```

Comment:
I'm not sure I agree that "properly creating a shared library" (note I don't mean an LCM binding library, I mean *any* portable shared library) is an "advanced" topic. (I should note also that it wasn't my intention for this document to be a tutorial on CMake itself; i.e. I assume the user is at least somewhat familiar with CMake already.) However, that being said, what I think might be useful would be to separate a bare bones C-and-C++-only bindings example (that explicitly builds a static library) as a "basic" tutorial and the current shared-C-library with Java and Python as an "advanced" tutorial. (This might also help toward a comment from @tkoolen in #105.) What do you think?